tests/playwright-test/reporter-html.spec.ts

```diff
@@ -825,10 +825,30 @@ for (const useIntermediateMergeReport of [true, false] as const) {
       expect(result.passed).toBe(1);
 
       await showReport();
-      await page.click('text=annotated test');
+      await page.getByRole('link', { name: 'annotated test' }).click();
       await expect(page.locator('.test-case-annotation')).toHaveText('issue: I am not interested in this test');
     });
 
+    test('should not crash on falsy non-string annotation', async ({ runInlineTest, page, showReport }) => {
```

Comment:
When adding a new test, please add an "issue" annotation whenever available.

Reply:
How ironic that I forget the annotation on a PR about annotations